### PR TITLE
update numpy

### PR DESCRIPTION
**Pull request by dcherian**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

@conda-forge-admin, please rerender

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ requirements:
     - setuptools_scm_git_archive
   run:
     - importlib-metadata
-    - numpy
+    - numpy >= 1.20
```
Comment (dcherian) [inline recipe/meta.yaml:26]:
```suggestion
    - numpy >=1.20
```

**Comment by dcherian** [comment]
@ocefpaf I forgot to bump numpy min version on the latest release. Is this enough or do we have explicitly remove the old builds somehow?

**Comment by ocefpaf** [comment]
For micromamba, mamba, and conda with libmamba this is enough b/c the higher build numbers are preferred. For vanilla conda one can get the lower build number with the bad pin depending on the solver's solution.

You can mark it as broken or send a repodata patch to fix it. The former is easier though.

**Comment by dcherian** [comment]
> mark it as broken

OK let's do this. Shall I follow the first half of the instructions in https://conda-forge.org/docs/maintainer/updating_pkgs.html#removing-broken-packages? (since the metadata is broken not the package itself)

**Comment by ocefpaf** [comment]
> Shall I follow the first half of the instructions in https://conda-forge.org/docs/maintainer/updating_pkgs.html#removing-broken-packages? (since the metadata is broken not the package itself)

Yep. That should do it.

**Pull request merged**